GitHubExplorer/GitHubExplorer/Coordinators/Coordinator.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Coordinator.swift
+//  GitHubExplorer
+//
+//  Created by ts51 on 6.07.20.
+//  Copyright © 2020 example. All rights reserved.
+//
+
+import UIKit
+
+protocol Coordinator {
+    var childCoordinators: [Coordinator] { get set }
```

Comment:
We don't have an actual need for child coordinators (for now), so this var seems unnecessary.

Reply:
Leaving as per tutorial recommendation (and, in this case, it's left empty)